app/services/testdata/RegisteredStatusGenerator.scala

```diff
@@ -39,9 +39,9 @@ trait RegisteredStatusGenerator extends BaseGenerator {
 
   def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier, rh: RequestHeader) = {
     val firstName = generatorConfig.firstName.getOrElse(Random.getFirstname(generationId))
-    val lastName = generatorConfig.lastName.getOrElse(Random.getLastname(generationId + 4000))
+    val lastName = generatorConfig.lastName.getOrElse(Random.getLastname(generationId))
     val preferredName = generatorConfig.preferredName.getOrElse(s"Pref$firstName")
-    val email = s"${generatorConfig.emailPrefix}${generationId + 4000}@mailinator.com"
+    val email = s"${generatorConfig.emailPrefix.getOrElse(Random.number())}a${generationId}@mailinator.com"
```

Comment:
So new format email addresses will be "100000a1@mailinator.com" ?

I think we should perhaps inject a constant string at the front to make them "unique" to us - that way we won't accidentally cross over with anyone else's mailinator account/accidentally send email to someone who isn't us